utils/getScenarioContent.tsx

```diff
@@ -0,0 +1,98 @@
+import { Claim, ClaimDetailsContent, ClaimStatusContent, ScenarioContent } from '../types/common'
+
+export enum ScenarioType {
+  PendingDetermination = 'Pending determination scenario',
+  BasePending = 'Base state with pending weeks',
+  BaseNoPending = 'Base state with no pending weeks',
+}
+
+/**
+ * Identify the correct scenario to display.
+ *
+ * @param {Qbject} claim
+ * @returns {Object}
+ */
+export function getScenario(claimData: Claim): ScenarioType {
+  // The pending determination scenario: if claimData contains any pendingDetermination
+  // objects
+  // @TODO: refactor with more detailed pending determination scenarios #252
+  if (claimData.pendingDetermination && claimData.pendingDetermination.length > 0) {
+    return ScenarioType.PendingDetermination
+  }
+  // The generic pending scenario: if there are no pendingDetermination objects
+  // AND hasPendingWeeks is true
+  else if (claimData.hasPendingWeeks === true) {
+    return ScenarioType.BasePending
+  }
+  // The generic "all clear"/base state scenario: if there are no pendingDetermination objects
+  // and hasPendingWeeks is false
+  else if (claimData.hasPendingWeeks === false) {
+    return ScenarioType.BaseNoPending
+  }
+  // This is unexpected
+  // @TODO: Log the scenario and display 500
+  else {
+    // throw new Error('Unexpected scenario')
+    return ScenarioType.BaseNoPending
+  }
```

Comment:
Note: This is temporary. In prod, this should result in 500, but I haven't yet resolved how we want to handle this during testing/development.